framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -694,7 +751,7 @@ private static void assertFilesExist(final List<File> expectedFiles) {
                 missingAbsoluteFilenames.add(missingFile.getAbsolutePath());
             }
             throw new RuntimeException(
-                    "The following files could not be located: "
+                    "The following files and/or directory could not be located: "
```

Comment:
`files` is plural, but `directory` is singular.

Reply:
There can only be one directory missing like `jdk11`/`jdk12` but there can be more than one files that could be missing.

Reply:
That assumes a certain use of that method, which is not documented in the javadoc.
I think in general it reads nicer to be consistent about singular/plural. That is, you could check the length of `missingAbsoluteFilenames` and if it's one use singular and otherwise plural. Or consistently use singular/plural, regardless of how many files or directories are not found.